src/test/groovy/com/meltwater/docker/compose/DockerComposeTest.groovy

```diff
@@ -1,6 +1,6 @@
 package com.meltwater.docker.compose
```

Comment:
Interesting fact: This test went from about 9 seconds to 3 seconds runtime using docker-compose v2.